src/copairs/map/filter.py

```diff
@@ -6,13 +6,38 @@
 import pandas as pd
 
 
-def validate_pipeline_input(meta, feats, columns):
+def validate_pipeline_input(
+    meta: pd.DataFrame, feats: np.ndarray, columns: List[str]
+) -> None:
+    """Validate the metadata and features for consistency and completeness.
+
+    Parameters:
+    ----------
+    meta : pd.DataFrame
+        The metadata DataFrame describing the profiles.
+    feats : np.ndarray
+        The feature matrix where rows correspond to profiles in the metadata.
+    columns : List[str]
+        List of column names in the metadata to validate for null values.
+
+    Raises:
+    -------
+    ValueError:
+        - If any of the specified metadata columns contain null values.
+        - If the number of rows in the metadata and features are not equal.
+        - If the feature matrix contains null values.
+    """
+    # Check for null values in the specified metadata columns
     if meta[columns].isna().any(axis=None):
-        raise ValueError("metadata columns should not have null values.")
+        raise ValueError("Metadata columns should not contain null values.")
+
+    # Check if the number of rows in metadata matches the feature matrix
     if len(meta) != len(feats):
-        raise ValueError("meta and feats have different number of rows")
+        raise ValueError("Metadata and features must have the same number of rows.")
+
+    # Check for null values in the feature matrix
     if np.isnan(feats).any():
-        raise ValueError("features should not have null values.")
+        raise ValueError("Features should not contain null values.")
```

Comment:
@axiomcura test are currently failing because edits of error messages are not synced with [those in tests](https://github.com/cytomining/copairs/blob/d318ed6cc962126b25fc9b6c68cd14df169e1806/tests/test_map.py#L172).

Reply:
Ahhh I see, I'll get on it right now! Thank you!